openmetadata-docs/content/deployment/security/ldap/index.md

```diff
@@ -66,11 +65,22 @@ Please see the below image for a sample LDAP Configuration in ApacheDS.
 Advanced LDAP Specific Configuration (Optional):
 
 - `maxPoolSize`: Connection Pool Size to use to connect to LDAP Server.
-- `sslEnabled`: Set to true if the SSL is enable to connecto to LDAP Server.
-- `keyStorePath`: Path of Keystore in case the sslEnabled is set to true.
-- `keyStorePassword`: Truststore Password.
-- `truststoreFormat`: TrustStore Format (Example :- JKS).
-- `verifyCertificateHostname`: Controls using TrustAllSSLSocketVerifier vs HostNameSSLSocketVerifier. In case the certificate contains cn=hostname of the Ldap Server set it to true.
+- `sslEnabled`: Set to true if the SSL is enable to connect to LDAP Server.
+- `truststoreConfigType`: Truststore type. Can select from {CustomTrustStore, HostName, JVMDefault, TrustAll}
+- `trustStoreConfig`: Config for the selected truststore type.
```

Comment:
please mention one of these is required , and the config will differ based on that

Reply:
addressed in the latest commit